src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -165,9 +88,11 @@ def get_mdtest_job_manager_command(self, manager):
             mpio_util = MpioUtils()
             if mpio_util.mpich_installed(self.hostlist_clients) is False:
                 self.fail("Exiting Test: Mpich not installed")
-            return Mpirun(self.mdtest_cmd, mpitype="mpich")
+            self.job_manager = Mpirun(self.mdtest_cmd, mpitype="mpich")
+        else:
+            self.job_manager = Orterun(self.mdtest_cmd)
```

Comment:
(pylint-redefined-variable-type) Redefinition of self.job_manager type from job_manager_utils.Mpirun to job_manager_utils.Orterun